### PR TITLE
Fix #308: refer to MLContext validation steps from MLGraphBuilder

### LINKED ISSUE #308: Validate MLContext
We need steps to validate an `MLContext` object.
PR in the works.

### PR DESCRIPTION
**Pull request by zolkis**
Fix #308: refer to MLContext validation steps from MLGraphBuilder constructor

Minor change, adds reference to missing algorithm.

@anssiko PTAL

***
<a href="https://pr-preview.s3.amazonaws.com/zolkis/webnn/pull/332.html" title="Last updated on Jan 24, 2023, 9:56 AM UTC (6cefee4)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/webmachinelearning/webnn/332/019a1f6...zolkis:6cefee4.html" title="Last updated on Jan 24, 2023, 9:56 AM UTC (6cefee4)">Diff</a>

### CODE REVIEW

**Comment by zolkis** [comment]
Looks like my editor stumbled upon line endings (trailing spaces) here as well.
Fortunately the diff is usable.

File: index.bs [hunk 1]
```diff
@@ -1019,11 +1019,9 @@ Both {{MLGraphBuilder}}.{{MLGraphBuilder/build()}} and {{MLGraphBuilder}}.{{MLGr
 The [=new=] {{MLGraphBuilder}} constructor steps are:
 1. If [=this=]'s [=relevant global object=]'s [=associated Document=] is not [=allowed to use=] the [=webnn-feature|webnn=] feature, throw a "{{SecurityError}}" {{DOMException}} and abort these steps.
 1. Let |context| be the first argument.
-1. If |context| is not a valid {{MLContext}}, throw a "{{TypeError}}" and abort these steps.
+1.  If the <a>validate MLContext</a> steps given |context| return `false`, throw a "{{TypeError}}" and abort these steps.
```
Comment (anssiko) [inline index.bs:1022]: Remove the _leading_ whitespace :-)

```suggestion
1. If the <a>validate MLContext</a> steps given |context| return `false`, throw a "{{TypeError}}" and abort these steps.

```

**Review by anssiko — CHANGES_REQUESTED** [review]
Thanks!

Protip: you can hide whitespace in GH diffs with `w=1` e.g. https://github.com/webmachinelearning/webnn/pull/332/files?diff=unified&w=1

**Review by anssiko — APPROVED** [review]

**Comment by anssiko** [comment]
Thanks @zolkis!

I'll merge this minor change to allow the WG to focus on additional work ahead.

**Pull request merged**